woocommerce/woocommerce-gutenberg-products-block
# Add missing selectors to data store documentation

## What

Fixes #11340

## Why

The Selectors of various data stores were not documented.  With this PR, we're adding missing selectors to data store documentation. 

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Read the documentation and check for typos, grammar errors, etc.
2. Check all the links work.



## Screenshots or screencast

N/A

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Skipping
